maps/tether/tether.dm

```diff
@@ -21,7 +21,9 @@
 	#include "tether-08-ships.dmm"
 	#include "tether-09-empty-surface.dmm"
 	#include "tether-10-empty-space.dmm"
-//	#include "tether-11-wild-surface.dmm" // Wilderness stuff removed until mobs can be optimized better.
+	#include "tether-11-wild-surface.dmm"
+	#include "tether-12-wild-crash-alt.dmm"
+	#include "tether-13-wild-temple.dmm"
```

Comment:
For now, why not remove these #includes and the tether-01 change and any other changes that enable the maps, and we can merge the rest of the other changes/fixes in. Then Leshana and I or anyone can work on optimizing the large startup time penalty and map gen stuff. In fact, it can probably be optimized using the new /tg/ maploader.

Reply:
tomorrow I make the changes then